core/gui/src/app/workspace/component/menu/menu.component.ts

```diff
@@ -527,6 +527,30 @@ export class MenuComponent implements OnInit {
     this.persistWorkflow();
   }
 
+  cloneVersion() {
+    const vid = this.workflowVersionService.selectedVersionId.getValue();
+    const selectedDisplayedVersionId = this.workflowVersionService.selectedDisplayedVersionId.getValue();
+    if (vid != null && selectedDisplayedVersionId != null) {
+      this.workflowVersionService
+        .cloneWorkflowVersion(vid, selectedDisplayedVersionId)
```

Comment:
Since the `WorkflowVersionService` already has a member variable of the version id and displayed version id, you don't need to pass them as arguments of `cloneWorkflowVersion`. Please remove the parameters and remove unnecessary getters.